packages/data-stores/src/reader/queries/use-subscription-manager-user-settings-query.ts

```diff
@@ -0,0 +1,33 @@
+import { useSelect } from '@wordpress/data';
+import { useQuery } from 'react-query';
+import { register as registerUserStore } from '../../user';
+import { fetchFromApi } from '../helpers';
+import type { SubscriptionManagerUserSettings } from '../types';
+
+// Credentials are only needed to create users
+const USER_STORE = registerUserStore( { client_id: '', client_secret: '' } );
+
+type EmailSettingsAPIResponse = {
+	settings: SubscriptionManagerUserSettings;
+};
+
+const useSubscriptionManagerUserSettingsQuery = () => {
+	const isLoggedIn = useSelect( ( select ) =>
```

Comment:
I would create a helper hook out of this (e.g. `useCheckUserLoggedIn`). We will need it for other API calls too, right?